plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -285,7 +285,7 @@ private function init_feature_definitions(): void {
 				'description'        => sprintf(
 					// translators: %s is the URL to the rate limiting documentation.
 					__( 'Enables rate limiting for Checkout place order and Store API /checkout endpoint. To further control this, refer to <a href="%s" target="_blank">rate limiting documentation</a>.', 'woocommerce' ),
-					'https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/StoreApi/docs/rate-limiting.md'
+					'https://developer.woocommerce.com/docs/apis/store-api/rate-limiting/'
```

Comment:
Why did you choose to link to the docs site instead of the markdown file in this case?

Reply:
I assumed translators would be working through the translations web interface rather than in the codebase, like the other example. But happy to send both links to the repo if that makes more sense.